data/systemd/Makefile

```diff
@@ -0,0 +1,15 @@
+SNAPD_ENVIRONMENT_FILE := /etc/environment
+SNAP_MOUNTDIR := /snap
+BINDIR := /usr/bin
+LIBEXECDIR := /usr/lib
+
+all:
```

Comment:
If we're going to use a Makefile, I would rather see it take a `DESTDIR` parameter and install the generated units to the correct place.

Add an `install` target, and have it write the real units to `SYSTEMDSYSTEMUNITDIR` (matching autotools name for this). Since you're doing the Ubuntu thing by default, this would be defined as `/lib/systemd/system`, whereas on RPM systems, I would pass in `%{_unitdir}` to ensure it went to the right place.